queries/idris/highlights.scm

```diff
@@ -0,0 +1,208 @@
+; ------------------------------------------------------------------------------
+; Literals and comments
+(integer) @number
+
+(literal
+  (number)) @number.float
+
+(char) @character
+
+[
+  (string)
+  (pat_string)
+  (triple_quote_string)
```

Comment:
`@string.documentation`?

Reply:
@srghma I think he meant only for `(triple_quote_string)` to be `@string.documentation`, sorry rest should be just `@string` :)

Actually, I think they should all be just `@string`. Correct me if I'm wrong but looks like inline documentation is delegated to `|||` comments?